backend/services/userService.js

```diff
@@ -250,4 +290,41 @@ const createPersonalDetails = async (req, res) => {
   }
 };
 
-export { createUser, authenticate, updateUserInfo, assignRole, assignManager, getUsers, createPersonalDetails };
+const createEmployeeDetails = async (req, res) => {
```

Comment:
If model name is EmploymentDetails won't it be better to call `createEmployeeDetails` to `createEmploymentDetails`.